tests/Docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
```

Comment:
Please keep docker-compose files in the root directory

Reply:
Why?

Reply:
@xepozz these are test-only images, not images intended for production use. I'd prefer having a single image for prod/test/dev but since it's not that, I think it makes sense to keep it in `tests`.

Reply:
No need to keep them in `tests` or somewhere else because it's unclear if you want to clone the project and run tests.
There won't be any production-like docker images because it's not a product to run it in production, it's a dependency.
Furthermore, it's also docker-compose for development.